BLE_GattServer/main.cpp

```diff
@@ -0,0 +1,668 @@
+#include <stdio.h>
+
+#include "platform/Callback.h"
+#include "events/Eventqueue.h"
+#include "platform/NonCopyable.h"
+
+#include "ble/BLE.h"
+#include "ble/Gap.h"
+#include "ble/GattClient.h"
+#include "ble/GapAdvertisingParams.h"
+#include "ble/GapAdvertisingData.h"
+#include "ble/GattServer.h"
+
+using mbed::callback;
+
+/**
+ * A Clock service that demonstrate the GattServer features.
+ *
+ * The clock service host three characteristics that respectively model the
+ * current hour, minute and second. The value of the second characteristic is
+ * incremented automatically by the system.
+ *
+ * When the value of the second characteristic reach 60, it is reset to 0 and
```

Comment:
I think the explanation is redundant. Simply noting that the characteristics model a clock and are automatically incremented is enough.